src/components/aeAddressInput/aeAddressInput.vue

```diff
@@ -8,7 +8,6 @@
     v-remove-spaces-on-copy
   />
 </template>
-
```

Comment:
It was intentional, I don't agree.

Reply:
What was intentional? The line break?

Reply:
@sadiqevani yes

Reply:
I don't understand this review...is it about the linebreak or hex colors?